src/test/java/seedu/duke/command/misc/AddCommandTest.java

```diff
@@ -0,0 +1,28 @@
+package seedu.duke.command.misc;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.DukeSession;
+import seedu.duke.Ingredient;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.duke.command.misc.AddCommand.isInList;
+
+class AddCommandTest {
+    @Test
+    public void itemIsInList() {
+        DukeSession dukeSession = new DukeSession();
+        Ingredient ingredient = new Ingredient("apple", 2);
+        DukeSession.Ingredients.add(ingredient);
+        assertTrue(isInList("apple"));
+    }
+
+    @Test
+    public void itemIsNotInList() {
+        DukeSession dukeSession = new DukeSession();
+        Ingredient ingredient = new Ingredient("orange", 2);
+        DukeSession.Ingredients.add(ingredient);
+        assertFalse(isInList("apple"));
+    }
```

Comment:
These two tests could be merged together (i.e. test both is and not in list for a single session)